kalite/coachreports/views.py

```diff
@@ -154,8 +154,9 @@ def tabular_view(request, report_type="exercise"):
     # Define how students are ordered--used to be as efficient as possible.
     student_ordering = ["last_name", "first_name", "username"]
 
-    # Get a list of topics (sorted) and groups
-    topics = [get_node_cache("Topic", language=language).get(tid["id"]) for tid in get_knowledgemap_topics(language=language) if report_type.title() in tid["contains"]]
+    # Get a list of topics that are at the third level down of the topic tree to use as items in the drop down.
```

Comment:
Needless to say, it's pretty cryptic. I guess this whole app will soon be replaced, though?